### DISCUSSION TITLE
SCSS, Sass or Stylus?

### DISCUSSION THREAD

**Discussion by InfoLibre** [element-plus/element-plus#4296] [Q&A]
Hello,
I'm looking for a Bootstrap alternative. I'm interested by https://github.com/fomantic/fomantic-ui and perhaps Element+, I don't know. Is Element+ using https://stylus-lang.com?

**Comment by msidolphin** [comment]
We are using scss.

**Comment by InfoLibre** [comment]
Could you add a Stylus version, like https://github.com/fomantic/fomantic-ui ? (they propose CSS, Less, Sass versions and Stylus version will be added soon).

**Comment by InfoLibre** [comment]
I'm using https://quilljs.com and https://vueup.github.io/vue-quill/ in my project. They both use Stylus and I would like to have only one CSS Postprocessor to use.

**Comment by msidolphin** [comment]
You don't need to care what processor we use, please refer to the documentation

**Comment by InfoLibre** [comment]
In https://element-plus.org/en-US/guide/theming.html I see indeed you are using Sass. Adding Stylus and Less files would be interesting.

**Reply by msidolphin** [reply]
> In https://element-plus.org/en-US/guide/theming.html I see indeed you are using Sass. Adding Stylus and Less files would be interesting.

We also support using css var to customize the theme

https://element-plus.org/en-US/guide/theming.html#by-css-variable
